### PR TITLE
Fix acceptance tests for regressions from 7/28/2016 build

### PR DESCRIPTION
**Pull request by avulfson17**
Fixes pelias/pelias#390

### CODE REVIEW

File: test_cases/encoding.json [hunk 1]
```diff
@@ -44,6 +42,48 @@
...
+          {
+            "source": "geonames",
+            "name": "Chambéry",
+            "macroregion": "Rhône-Alpes",
+            "locality": "Chambéry",
+            "label": "Chambéry, France",
+            "country": "France"
+          }
+        ]
+      }
```
Comment (orangejulius) [inline test_cases/encoding.json]: This whole section is excellent, but should be in a separate pull request :)

File: test_cases/brooklyn.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
       "type": "dev",
       "in": {
         "text": "brooklyn",
-        "layers": "locality,localadmin",
+        "layers": "borough",
```
Comment (orangejulius) [inline test_cases/brooklyn.json]: Two of these four tests are now identical to the other two, and can just be removed instead of being updated.

File: test_cases/search.json [hunk 3]
```diff
@@ -156,7 +156,7 @@
     },
     {
       "id": 7,
-      "status": "pass",
+      "status": "fail",
```
Comment (orangejulius) [inline test_cases/search.json]: We will definitely want to add a "description" field here that describes the problem, as well as an "issue" field that links to either a Pelias issue we create, or one in the [Openaddresses](https://github.com/openaddresses/openaddresses/issues) repo if one is there (we should report it there actually). Let's figure out which issue(s) should be made and then add it here.

**Comment by orangejulius** [comment]
Nice! LGTM

**Comment by orangejulius** [comment]
I love the fact that we now get the :white_check_mark: from Travis that this PR fixes all the failing tests :)

**Comment by dianashk** [comment]
That ✅  is pretty slick! Great work setting that up. :shipit:
